pontoon/messaging/views.py

```diff
@@ -257,10 +303,9 @@ def send_message(request):
                 description=f"{subject}<br/><br/>{body}",
                 identifier=identifier,
             )
+            notification_recipients_count += 1
 
-        log.info(
-            f"Notifications sent to the following {recipients.count()} users: {recipients.values_list('email', flat=True)}."
-        )
+        log.info(f"Notifications sent to {notification_recipients_count} users.")
```

Comment:
Why not just get the length directly?
```suggestion
        log.info(f"Notifications sent to {len(recipients)} users.")
```

Reply:
🤦